parser_test.go

```diff
@@ -1784,6 +1784,14 @@ func (s *testParserSuite) TestDDL(c *C) {
 		{"CREATE INDEX idx USING BTREE ON t (a) USING HASH COMMENT 'foo'", true, ""},
 		{"CREATE INDEX idx USING BTREE ON t (a)", true, ""},
 
+		//For dorp index statement
+		{"DROP INDEX `a` ON `t`", true, ""},
```

Comment:
the left string is test data, the right sting is expected text. test it like this:
```suggestion
		{"DROP INDEX a ON t", true, "DROP INDEX `a` ON `t`"},
```